frontend/src/common/pages/DocumentUnavailable.tsx

```diff
@@ -0,0 +1,17 @@
+import "./UniversalUnexpected.scss";
+import { ErrorPage } from "../components/error/ErrorPage";
+
+export const DocumentUnavailable = () => {
+  return (
+    <ErrorPage
+      errorTitle="Your document is on the way"
+      msgNode={
+        <div className="document-unavailable">
+        Your document is being created.
+        <br />
+        Please check again later.
+    </div>
```

Comment:
From lines 10-13, adjust the tab spacing to the right a bit to match the ```<div``` on line 9